parsec/core/gui/files_widget.py

```diff
@@ -628,13 +626,15 @@ def item_activated(self, file_type, file_name):
         elif file_type == FileType.Folder:
             self.load(self.current_directory / file_name)
 
-    def reload(self):
-        self.load(self.current_directory)
+    def schedule_reload(self):
+        self.load(self.current_directory, delay=self.RELOAD_FILES_LIST_DELAY)
```

Comment:
Renaming to `schedule_reload` is no longer justified.

Reply:
:+1: